packages/storage-drivers/resolve-storage-mongo/src/index.js

```diff
@@ -25,6 +28,12 @@ function createDriver({ url, collection }) {
                     coll
                         .createIndex('timestamp')
                         .then(() => coll.createIndex('aggregateId'))
+                        .then(() =>
+                            coll.createIndex(
```

Comment:
Should this change be reflected in related tests: https://github.com/reimagined/resolve/blob/aggregate_version/packages/storage-drivers/resolve-storage-mongo/test/index.tests.js ?

Reply:
ok